Marlin/timestamp_t.h

```diff
@@ -46,72 +46,82 @@ struct timestamp_t {
   }
 
   /**
-   * @brief Formats the date as number of years
+   * @brief Formats the timestamp in years
```

Comment:
It's not really a "timestamp" (a point in time) but rather a "duration" or an "interval." And this function doesn't really "format" the interval (a string operation). Rather, it returns the number of total years in the interval.